Update_user.py

```diff
@@ -0,0 +1,34 @@
+
+#!/usr/bin/python
+
+#Turn on debug mode.
+import cgitb
+cgitb.enable()
+
+#Print necessary headers.
+print("Content-Type: text/html; charset=utf-8\n\n")
+
+#Connect to the database.
+import cgi
+form = cgi.FieldStorage()
+if "player" not in form or "newplayer" not in form:
+  print "Error"
+else:
+  print ("Hello" form["player"].value)
+  
+#Connect to the database.
+import pymysql
+conn = pymysql.connect(
+  db = 'example',
+  user = 'root'
+  passwd = 'password' #changed this incase it matter
+  host = 'localhost')
+c = conn.cursor()
+
+#Insert some example data - Having some problem with this as my database says there is an error.
+c.execute("UPDATE players SET player.first_name = "newplayer" WHERE player.first_name = "player")
```

Comment:
I believe the error here is that you need to assign "player" and "newplayer" from the form (above) to a variable, which should be used in place of them here. 
Also, the variable would need to be concatenated to the string.